src/policy/sft.rs

```diff
@@ -130,6 +139,15 @@ impl SFT for EmptySpaceSFT {
         warn!("Object in empty space!");
         false
     }
+    fn pin_object(&self, _object: ObjectReference) -> bool {
```

Comment:
I think you should panic for these 3 new methods for `EmptySFT`. Basically, MMTk tries to pin/unpin/query for an object that does not belong to any space. The work cannot be done.